Integrations/ThreatQ/ThreatQ.py

```diff
@@ -0,0 +1,1248 @@
+        'Description': clean_html_from_string(data.get('description')),
+        'Source': sources_to_demisto_format(data.get('sources')),
+        'Attribute': attributes_to_demisto_format(data.get('attributes'))
+    }
+    return ret
+
+
+def file_data_to_demisto_format(data):
+    raw = {
+        'ID': data.get('id'),
+        'CreatedAt': data.get('created_at'),
+        'UpdatedAt': data.get('updated_at'),
+        'Size': data.get('file_size'),
+        'MD5': data.get('hash'),
+        'Type': TYPE_ID_TO_FILE_TYPE[data.get('type_id')],
+        'Name': data.get('name'),
+        'Title': data.get('title'),
+        'Description': data.get('description'),
+        'ContentType': content_type_to_demisto_format(data.get('content_type_id')),
+        'MalwareLocked': malware_locked_to_demisto_format(data.get('content_type_id')),
+        'Source': sources_to_demisto_format(data.get('sources')),
+        'Attribute': attributes_to_demisto_format(data.get('attributes'))
+    }
+
+    return raw
+
+
+def get_pivot_id(obj1_type, obj1_id, obj2_type, obj2_id):
+    # A pivot id represents a connection between two objects.
+
+    url_suffix = '/{0}/{1}/{2}'.format(OBJ_DIRECTORY[obj1_type], obj1_id, OBJ_DIRECTORY[obj2_type])
+    res = tq_request('GET', url_suffix)
+
+    for related_object in res['data']:  # res['data'] contains all the related objects of obj_id1
+        if int(related_object.get('id')) == int(obj2_id):
+            return int(related_object['pivot']['id'])
+
+
+def add_malicious_data(generic_context, tq_score):
+    generic_context['Malicious'] = {
+        'Vendor': 'ThreatQ',
+        'Description': 'Score from ThreatQ is {0}'.format(tq_score)
+    }
+
+
+def set_ioc_entry_context(ioc_type, raw, dbot, generic):
+    if dbot.get('Score') == 3:
+        add_malicious_data(generic, raw.get('TQScore', -1))
+    ec = {
+        outputPaths[ioc_type]: generic,
+        'DBotScore': dbot
+    }
+    if raw:
+        ec[CONTEXT_PATH['indicator']] = raw
+    return ec
+
+
+def build_readable_for_search_by_name(ioc_context, event_context, adversary_context, file_context):
+    if not (ioc_context or event_context or adversary_context or file_context):
+        return 'No results.'
+
+    human_readable = ''
+    if ioc_context:
+        human_readable += tableToMarkdown('Search Results - Indicators', ioc_context)
+    if event_context:
+        human_readable += tableToMarkdown('Search Results - Events', event_context)
+    if adversary_context:
+        human_readable += tableToMarkdown('Search Results - Adversaries', adversary_context)
+    if file_context:
+        human_readable += tableToMarkdown('Search Results - Files', file_context)
+
+    return human_readable
+
+
+def build_readable(readable_title, obj_type, data, dbot_score=None, metadata=None):
+    if isinstance(data, dict):  # One object data
+        data['DBotScore'] = dbot_score  # We add DBot Score data only for the readable output - then we pop it back
```

Comment:
Why?

Reply:
I don't know why it still appears here, [here](https://github.com/demisto/content/blob/ThreatQ-integration/Integrations/ThreatQ/ThreatQ.py) it doesn't